.editorconfig

```diff
@@ -0,0 +1,21 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
Please remove this file which seems to be specific to your local development environment